static/fortran-mode.js

```diff
@@ -0,0 +1,543 @@
+// This file has evolved from 
```

Comment:
I appreciate the sentiment, but I think it's more clear to just claim copyright for yourself, or "Compiler Explorer Authors", if you're comfortable with that.

Reply:
Also feel free to add yourself to the AUTHORS.md

Reply:
I'm a little lost here to be honest. I'm personally not terribly fuzzed about adding or keeping any copyright statement, but really have no experience with that. I also don't feel it was a substantial contribution. If anything apart from "Compiler Explorer Authors" is added (which it probably should due to the starting point), my preference would be to mention my employer as I see and did this in relation to my day-to-day work.

I like the project (and open-source in general), I even believe one of colleagues uses or at least recommends compiler-explorer during his courses. As Fortran was missing I gave it a shot since I don't consider it outdated or unused.

As for the `AUTHORS.md`, again it's really for you to judge. I don't think the few LOC warrant being mentioned therein. The small addition is anyways not for the main chunk of code.

Reply:
Would be ok/possible or even necessary to add the license header from the other file I took inspiration from back in? This did include a bit about 'substantial parts' in this other file, and all I copied was the existing list of Fortran keywords (which form the standard).

Reply:
Hi! We have now created a `CONTRIBUTORS.md` where I think you could add yourself instead of the `AUTHORS.md` file. :)

Reply:
That feels more appropriate, thanks. I guess this could be our organisation link on github instead of me and my name?